Dockerfile

```diff
@@ -40,12 +40,30 @@ RUN grep -aL "github.com" target/release/deps/* | xargs rm \
 COPY . .
 
 # Rebuild the agent
-RUN make build RELEASE=1
+RUN cargo build --release
 RUN strip target/release/logdna-agent
 
 # Use ubuntu as the final base image
 FROM ubuntu:18.04
 
+ARG REPO
+ARG BUILD_DATE
+ARG VCS_REF
+ARG VCS_URL
+ARG BUILD_VERSION
+
+LABEL org.label-schema.schema-version="1.0"
+LABEL org.label-schema.url="https://logdna.com"
+LABEL org.label-schema.build-date="${BUILD_DATE}"
+LABEL org.label-schema.maintainer="LogDNA <support@logdna.com>"
+LABEL org.label-schema.name="answerbook/${REPO}"
+LABEL org.label-schema.description="LogDNA Agent V2"
```

Comment:
Can we use the description we use on GitHub: `The blazingly fast, resource efficient log collection client`

Reply:
sure